packages/app/src/systems/Network/components/NetworkItem/NetworkItem.tsx

```diff
@@ -61,12 +61,12 @@ export const NetworkItem = forwardRef<HTMLDivElement, NetworkItemProps>(
         ref={ref}
         onClick={() => onPress?.(network)}
         css={{ ...styles.root, ...css }}
-        isActive={network.isSelected}
-        data-active={network.isSelected}
+        isActive={network?.isSelected}
```

Comment:
It should always have a network selected it was returning a issue here?

Reply:
yeah i think its related to typescript this was actually breaking the screen so i added this optional chaining to stop breaking